src/exchange/ExchangeHomeScreen.test.tsx

```diff
@@ -17,8 +16,22 @@ const exchangeRates: ExchangeRates = makeExchangeRates('0.11', '10')
 describe('ExchangeHomeScreen', () => {
   it('renders and behaves correctly for non CP-DOTO restricted countries', () => {
     const store = createMockStore({
-      goldToken: { balance: '2' },
-      stableToken: { balances: { [Currency.Dollar]: '10' } },
+      tokens: {
+        tokenBalances: {
+          [mockCusdAddress]: {
+            ...mockTokenBalances[mockCusdAddress],
+            balance: '10',
+          },
+          [mockCeloAddress]: {
+            ...mockTokenBalances[mockCeloAddress],
+            balance: '2',
+          },
+          [mockCeloAddress]: {
+            ...mockTokenBalances[mockCeloAddress],
+            balance: '2',
+          },
```

Comment:
It's set twice for Celo:
```suggestion
          [mockCeloAddress]: {
            ...mockTokenBalances[mockCeloAddress],
            balance: '2',
          },
```